README.md

```diff
@@ -29,9 +29,14 @@ The website *[filterblade.xyz](http://www.filterblade.xyz)* is the user-friendly
 
 2) Extract the files using WinRar/7zip or a similar tool.
 
-3) Paste the .filter files (or just the folder content, if you want to use a specific style, paste the files from one of the (STYLE) folders instead) into the following folder: *%userprofile%/Documents/My Games/Path of Exile*
-
-4) INGAME: Escape -> Options -> UI -> Scroll down -> Select one of the filters from the dropdown box.
+3) Paste the .filter files (or just the folder content, if you want to use a
+specific style, paste the files from one of the (STYLE) folders instead) into
+the following folder: ```%userprofile%/Documents/My Games/Path of Exile```.  On
+Linux, this directory will be within
+```~/.local/share/Steam/steamapps/compatdata/```.
+
+4) INGAME: Escape -> Options -> UI -> Scroll down -> Select one of the filters
```

Comment:
nit: Why the change on this line? The linebreak is not done on the other paragraphs above, so it's probably not worth changing it here